examples/src/routes/BasicComponents.jsx

```diff
@@ -81,6 +83,9 @@ export const BasicComponents = () => (
         <h2 id="treemap">Treemap</h2>
         <ExampleWithSource for={TreemapExample} source={TreemapExampleSRC} />
 
+        <h2 id="heatmap">Heatmap</h2>
```

Comment:
I am missing testcafe test for this element like others visualizations have

Reply:
Done